src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -19,17 +19,49 @@ const useSaveAnonChatHook = () => {
   const {goToChatScreen} = useChatUtilsHook();
   const {signedProfileId, anonProfileId} = useUserAuthHook();
 
-  const helperFindChatById = async (object: InitAnonymousChatData): Promise<ChannelListSchema> => {
+  const helperFindChatById = async (
+    object: InitAnonymousChatData
+  ): Promise<ChannelListSchema | null> => {
     const channelId = object?.message?.cid?.replace('messaging:', '');
     const channelList = await ChannelListSchema.getById(localDb, channelId);
 
     const channelListSchema = ChannelListSchema.fromDatabaseObject(channelList);
     return channelListSchema;
   };
 
+  const saveUserAndChannelListMember = async (channelId, member) => {
+    try {
+      const userMember = UserSchema.fromInitAnonymousChatAPI(member, channelId);
+      await userMember.saveOrUpdateIfExists(localDb);
+    } catch (e) {
+      console.log('error saveUserAndChannelListMember', e);
+    }
+  };
+
   const helperGoToAnonymousChat = async (object: InitAnonymousChatData, from: string) => {
-    const channelListSchema = await helperFindChatById(object);
-    goToChatScreen(channelListSchema, from);
+    try {
+      const channelListSchema = await helperFindChatById(object);
+      goToChatScreen(channelListSchema as ChannelList, from);
+    } catch (e) {
+      console.log('error helperGoToAnonymousChat', e);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the error handling in `helperGoToAnonymousChat` function could be improved. Simply logging the error to the console isn't very helpful in a production environment. Consider throwing the error after logging it or handle it in a way that doesn't fail silently.
```diff
-      console.log('error helperGoToAnonymousChat', e);
+      // log the error and then throw
+      console.error('error helperGoToAnonymousChat', e);
+      throw e;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->